best_practices/forms.rst

```diff
@@ -218,4 +218,22 @@ handling the form submit. For example, you *could* have a ``new()`` action that
 submit. Both those actions will be almost identical. So it's much simpler to let
 ``new()`` handle everything.
 
+.. versionadded:: 4.3
+   The ``AbstractController::isFormValid()`` method was added in Symfony 4.3.
+
+It's also possible to use a short-cut if the ``AbstractController`` is extended:
+
+.. code-block:: php
+
+    public function newAction(Request $request)
```

Comment:
I think it's better to follow best practices:
> Don't add the Action suffix to the methods of the controller actions.

Link: https://symfony.com/doc/current/best_practices/controllers.html#controller-action-naming

Reply:
You are totally right. I never add it in my own code :joy: I don't know why I did it there. 
Anyway, fixed